### PR TITLE
Add KCD Utrecht 2023

### PR DESCRIPTION
**Pull request by ams0**
Added our new event in The Netherlands with CFP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ All the data (past and coming) is available publicly in JSON:
 
 ### December
 
+* 1: [KCD Utrecht 2023](https://kcdutrecht.nl/) - Utrecht (The Netherlands) <a href="https://forms.gle/JyrTeEJuk2PVAT7d9"><img alt="KCD Utrecht" src="https://img.shields.io/static/v1?label=CFP&message=until%2022-October-2023&color=red"></a>
```
Comment (scraly) [inline README.md:213]:
```suggestion
* 1: [KCD Utrecht 2023](https://kcdutrecht.nl/) - Utrecht (The Netherlands) <a href="https://forms.gle/JyrTeEJuk2PVAT7d9"><img alt="KCD Utrecht" src="https://img.shields.io/static/v1?label=CFP&message=until%2022-October-2023&color=green"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks :) 

CFP is opened so the color of the badge is green ;)

File: README.md (hunk 1 above)
Comment (ams0) [reply README.md]: fixed in last commit, thanks!

**Comment by scraly** [comment]
thanks :)

**Pull request merged**